client/landing/stepper/hooks/use-verified-param.ts

```diff
@@ -0,0 +1,5 @@
+import { useQuery } from './use-query';
+
+export function useVerifiedParam(): string | null {
+	return useQuery().get( 'verified' );
```

Comment:
not sure if naming it emailVerified here as well will be a bit more descriptive and easier to follow along

Reply:
I'm going to rename it verifiedParam in other areas of code I modified, so it can be consistent with existing code related to email Verification notifications.